scripts/irods/test/test_iadmin.py

```diff
@@ -2132,6 +2132,10 @@ def test_moduser_type_rodsgroup(self):
                                    'STDERR', 'SYS_NOT_ALLOWED')
         self.assertEqual('rodsuser', lib.get_user_type(self.admin, self.username))
 
+    def test_moduser_bad_downgrade(self):
+        self.assertEqual('rodsadmin', lib.get_user_type(self.admin, 'rods'))
+        self.admin.assert_icommand(['iadmin', 'moduser', 'rods', 'type', 'rodsuser'], 'STDERR_SINGLELINE', 'SYS_NOT_ALLOWED')
```

Comment:
`SYS_NOT_ALLOWED` could appear in another string in the future which could lead to the test producing incorrect results.

Adjust the assertion so that `SYS_NOT_ALLOWED` can't lead to incorrect results.

Reply:
oh, you mean... make it more specific?   so that it catches *just* this particular error?

Reply:
Correct.